app/models/manageiq/providers/cisco_intersight/physical_infra_manager/event_catcher/stream.rb

```diff
@@ -31,7 +31,9 @@ def poll(&block)
               :filter  => "ModTime gt #{since}",
               # Use only selected attributes. Validation of other within the SDK lib might fail:
               :select  => '$select=CreateTime,ModTime,Name,Status,Email,WorkflowCtx,ClassId,Message',
-              :orderby => 'ModTime' # Orders elements in ascending order based on atribute ModTime
+              :orderby => 'ModTime', # Orders elements in ascending order based on atribute ModTime
+              :top     => $top=100, # Specifies the maximum number of resources to return in the response.
+              :skip    => $skip=0, # Specifies the number of resources to skip in the response.
```

Comment:
Hey @tjazsch sorry if I'm missing something obvious, but this looks like it is assigning 100 to a new global variable `$top` and then assigning it to the `:top` key of this hash.

Should this be?
```suggestion
              :top     => 100, # Specifies the maximum number of resources to return in the response.
              :skip    => 0, # Specifies the number of resources to skip in the response.
```

Reply:
Sorry, wasn't able to get to fixing the solution before. Thank you!